Resources/shell_integration/iterm2_shell_integration.xsh

```diff
@@ -0,0 +1,122 @@
+# This program is free software; you can redistribute it and/or
```

Comment:
Files in this directory are copied from https://github.com/gnachman/iTerm2-Shell-Integration, which is the source of truth for shell integration scripts. Please open a new PR in that repo and we'll need to merge it first.